rollbar-spring-boot-webmvc/src/main/java/com/rollbar/spring/boot/webmvc/RollbarSpringBootConfiguration.java

```diff
@@ -0,0 +1,31 @@
+package com.rollbar.spring.boot.webmvc;
+
+import com.rollbar.web.listener.RollbarRequestListener;
+
+import javax.servlet.ServletContext;
+
+import org.springframework.boot.web.servlet.ServletContextInitializer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class RollbarSpringBootConfiguration {
```

Comment:
I don't understand why we want to provide this `Configuration` in this way, that registers a bean by creating an anonymous class of a `ServletContextInitializer`.

I think we can just provide an implementation of a `ServletContextInitializer` that is the piece needed. Also, I think that if you use the `@Component` annotation it will get registered by using the `@ComponentScan({"com.rollbar.spring"})`